packages/core/src/components/alert/alert.scss

```diff
@@ -0,0 +1,104 @@
+:host {
+  display: block;
+
```

Comment:
Add `box-sizing: border-box`. The ShadowDOM prevents global styles from leaking through.